packages/@lightningjs/ui-components/src/components/TabBar/Tab.js

```diff
@@ -116,12 +123,13 @@ export default class Tab extends Surface {
       y: this._Content.h / 2
     };
     if (this.icon) {
-      textPatch.mountX = 0;
       textPatch.x = this._iconW + this.style.iconMarginRight;
+      textPatch.mountX = 0;
     } else {
-      textPatch.mountX = 0.5;
       textPatch.x = this._Content.w / 2;
+      textPatch.mountX = 0.5;
```

Comment:
Does this ordering actually matter? I'm a little surprising we wouldn't want it to be the original order, so just curious